package.json

```diff
@@ -16,6 +16,7 @@
     "jwt-encode": "^1.0.1",
     "miragejs": "^0.1.47",
     "react": "^18.2.0",
+    "react-css-modules": "^4.7.11",
```

Comment:
react-css-moules is an additional package, which is not needed for CSS modules. 
While it may be useful to some, we would want to keep this template as raw as possible so people can customize on their own
Remove it and modules should work regardless, lmk if you face any problems